### DISCUSSION TITLE
Adding a new Product Attribute

### DISCUSSION THREAD

**Discussion by sgwcollins** [saleor/saleor#10430] [Q&A]
Hey i love the platform, i am looking for the best way to add a Product Attribute.

We are trying to add different pricing options for the same product.

In this use case should we make the changes to the core saleor app, or create a plugin

**Comment by peelar** [comment]
Hello @sgwcollins 👋 

The answer varies on your use case. Could you describe it in more detail? What is it that you are trying to achieve?

In the meantime, there are two things I can send over to you right away:
- [the `productAttributeAssign` mutation](https://docs.saleor.io/docs/3.x/developer/api-reference/mutations/product-attribute-assign)
- and to give you some better context, our [Prices tutorial](https://learn.saleor.io/prices/pricing-range/)
